backend/src/test/java/com/sim_backend/websockets/observers/AuthorizeObserverTest.java

```diff
@@ -0,0 +1,44 @@
+package com.sim_backend.websockets.observers;
+
+import static org.mockito.Mockito.*;
+
+import com.sim_backend.websockets.events.OnOCPPMessage;
+import com.sim_backend.websockets.messages.AuthorizeResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.*;
+
+public class AuthorizeObserverTest {
+  @Mock private OnOCPPMessage testMessage;
+
+  private AuthorizeObserver testObserver;
+
+  @BeforeEach
+  void setUp() {
+    MockitoAnnotations.openMocks(this);
+    testObserver = new AuthorizeObserver();
+  }
+
+  @Test
+  void testAuthorizeMessage() {
```

Comment:
Capture output from `onMessageReceived` and assert that it matches the expected message. You can get a stream with something like
`private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();`
 `System.setOut(new PrintStream(outContent));`
from the java.io.ByteArrayOutputStream, and java.io.PrintStream libraries.